macos/install_all.sh

```diff
@@ -2,5 +2,5 @@
 source ../common/arch.sh
 
 for arch in $ARCHES; do
-brew install binutils-$arch.rb
-done
\ No newline at end of file
+brew upgrade binutils-$arch.rb
```

Comment:
Homebrew does not install formulae with `upgrade` command
```suggestion
brew install binutils-$arch.rb
```